pkg/node/node_controller.go

```diff
@@ -54,7 +55,11 @@ func (b *DirectCSIDriveListener) Delete(ctx context.Context, obj *v1alpha1.Direc
 }
 
 func startController(ctx context.Context) error {
-	ctrl, err := listener.NewDefaultDirectCSIController("node-controller", "leader-lock", 40)
+	hostname, err := os.Hostname()
+	if err != nil {
+		return err
+	}
+	ctrl, err := listener.NewDefaultDirectCSIController("node-controller", hostname, 40)
```

Comment:
this is to ensure that controllers from different nodes act as independent controllers, instead of a combined unit with a leader